CONTRIBUTING.md

```diff
@@ -1,95 +1,111 @@
 # Contribution Guidelines
+
 This document contains the rules and guidelines that developers are expected to follow, while contributing to this repository.
 
-* All contributions must NOT add any SwiftLint warnings or errors. There is a GitHub action setup for any PRs to `dev`, and Xcode will show any warnings/errors.
+- All contributions must NOT add any SwiftLint warnings or errors. There is a GitHub action setup for any PRs to `dev`, and Xcode will show any warnings/errors.
```

Comment:
Don't change this a `*` or `-` works to create bullet points, and because everything already uses `*` use that